<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageMasterGeneral</class>
 <widget class="QWidget" name="PageMasterGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabShape">
      <enum>QTabWidget::Triangular</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="PowerState">
      <attribute name="title">
       <string>PowerState</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="ParamTable" name="powerStateTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CAN">
      <attribute name="title">
       <string>CAN</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="ParamTable" name="canTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="TemperatureSensors">
      <attribute name="title">
       <string>Sensors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="ParamTable" name="masterSensorsTab"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ParamTable</class>
   <extends>QTableWidget</extends>
   <header>widgets/paramtable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
